core/src/test/java/com/taobao/arthas/core/util/StringUtilsTest.java

```diff
@@ -18,4 +27,388 @@ public void testHumanReadableByteCount() {
+        Assert.assertEquals("", StringUtils.trimLeadingCharacter("", 'a'));
+        Assert.assertEquals("foo", StringUtils.trimLeadingCharacter("foo", 'a'));
+        Assert.assertEquals("", StringUtils.trimLeadingCharacter("a", 'a'));
+        Assert.assertEquals("foo", StringUtils.trimLeadingCharacter("afoo", 'a'));
+    }
+
+    @Test
+    public void testTrimLeadingWhitespace() {
+        Assert.assertEquals("", StringUtils.trimLeadingWhitespace(""));
+        Assert.assertEquals("", StringUtils.trimLeadingWhitespace(" "));
+        Assert.assertEquals("!", StringUtils.trimLeadingWhitespace("!"));
+        Assert.assertEquals("foo  ", StringUtils.trimLeadingWhitespace("  foo  "));
+    }
+
+    @Test
+    public void testTrimTrailingCharacter() {
+        Assert.assertEquals("foo", StringUtils.trimTrailingCharacter("foo!", '!'));
+        Assert.assertEquals("", StringUtils.trimTrailingCharacter("", '!'));
+        Assert.assertEquals("", StringUtils.trimTrailingCharacter("!", '!'));
+    }
+
+    @Test
+    public void testTrimTrailingWhitespace() {
+        Assert.assertEquals("", StringUtils.trimTrailingWhitespace(" "));
+        Assert.assertEquals("", StringUtils.trimTrailingWhitespace(""));
+        Assert.assertEquals("  foo", StringUtils.trimTrailingWhitespace("  foo   "));
+    }
+
+    @Test
+    public void testTrimWhitespace() {
+        Assert.assertEquals("\u6d7c\u1280", StringUtils.trimWhitespace(" \u6d7c\u1280\u1680"));
+        Assert.assertEquals("", StringUtils.trimWhitespace(" "));
+        Assert.assertEquals("(", StringUtils.trimWhitespace("("));
+        Assert.assertEquals("", StringUtils.trimWhitespace(""));
+        Assert.assertEquals("foo foo", StringUtils.trimWhitespace(" foo foo "));
+    }
+
+    @Test
+    public void testUncapitalize() {
+        Assert.assertEquals("a", StringUtils.uncapitalize("A"));
+        Assert.assertEquals("a", StringUtils.uncapitalize("a"));
+        Assert.assertEquals("", StringUtils.uncapitalize(""));
+    }
+
+    @Test
+    public void testCapitalize() {
+        Assert.assertEquals("A", StringUtils.capitalize("a"));
+        Assert.assertEquals("A", StringUtils.capitalize("A"));
+        Assert.assertEquals("", StringUtils.capitalize(""));
+    }
+
+    @Test
+    public void testUnqualify() {
+        Assert.assertEquals("/", StringUtils.unqualify("....../"));
+        Assert.assertEquals(" ", StringUtils.unqualify("!!!!!! ", '!'));
+    }
+
+    @Test
+    public void testWrap() {
+        Assert.assertEquals("\n!", StringUtils.wrap("!", 0));
+        Assert.assertEquals("", StringUtils.wrap("", 1));
+        Assert.assertEquals("f\no\no", StringUtils.wrap("foo", 1));
+        Assert.assertEquals("f\n", StringUtils.wrap("f\n", 1));
+        Assert.assertEquals("\u0001\n", StringUtils.wrap("\u0001\n", 3));
+    }
+
+    @Test
+    public void testClassLoaderHash() {
+        Assert.assertEquals("null", StringUtils.classLoaderHash(null));
```

Comment:
@JohnLBergqvist had to change this back as the test failed when it use `assertnull`